src/pages/home/_what-our-clients-say.js

```diff
@@ -343,7 +343,7 @@ const WhatOurClientsSay = () => {
                         </TrustPilotWidget>
                     </ClientCard>
                     <ClientCard position="relative" direction="column">
-                        <QuoteIcon src={Quote} />
+                        <QuoteIcon src={Quote} alt="quote icons" />
```

Comment:
If it's a purely visual enhancer icon, we can omit the alt and leave it as an empty string for better accessibility. But if the icon has textual information then we should add the alt.